targets/targets.json

```diff
@@ -706,6 +707,7 @@
         "inherits": ["Target"],
         "detect_code": ["0311"],
         "device_has": ["USTICKER", "LPTICKER", "RTC", "ANALOGIN", "ANALOGOUT", "EMAC", "I2C", "I2CSLAVE", "INTERRUPTIN", "PORTIN", "PORTINOUT", "PORTOUT", "PWMOUT", "SERIAL", "SERIAL_FC", "SLEEP", "SPI", "SPISLAVE", "STDIO_MESSAGES", "TRNG", "FLASH"],
+        "features": ["STORAGE"],
```

Comment:
If you just want to add default SD to K66F then you don't really need to add feature STORAGE. Feature STORAGE will add the cfstore. Do you really need it?

Reply:
this is not needed - @maclobdell Please review